GEOSldas_App/LDASsa_DEFAULT_inputs_ensupd.nml

```diff
@@ -2363,9 +2363,87 @@ obs_param_nml(53)%xcorr          = 0.
 obs_param_nml(53)%ycorr          = 0.
 obs_param_nml(53)%adapt          = 0
 
+! --------------------------------------------------------------------------------------------------------------
+!
+!  54 = CYGNSS_SM (CYGNSS Level 3 soil moisture version 3.2)
+!
+! https://podaac.jpl.nasa.gov/dataset/CYGNSS_L3_SOIL_MOISTURE_V3.2
+
+obs_param_nml(54)%descr          = 'CYGNSS_SM'
+obs_param_nml(54)%orbit          = 3
+obs_param_nml(54)%pol            = 0
+obs_param_nml(54)%N_ang          = 0
+obs_param_nml(54)%freq           = 0
+obs_param_nml(54)%FOV            = 20.
+obs_param_nml(54)%FOV_units      = 'km'
+obs_param_nml(54)%assim          = .false.
+obs_param_nml(54)%scale          = .false.
+obs_param_nml(54)%getinnov       = .false.
+obs_param_nml(54)%RTM_ID         = 0
+obs_param_nml(54)%bias_Npar      = 0
+obs_param_nml(54)%bias_trel      = 864000
+obs_param_nml(54)%bias_tcut      = 432000
+obs_param_nml(54)%nodata         = -9999.
+obs_param_nml(54)%varname        = 'sfmc'
+obs_param_nml(54)%units          = 'm3 m-3'
+obs_param_nml(54)%path           = '/discover/nobackup/amfox/CYGNSS_obs/'
```

Comment:
Let's see if we can put the data on a project disk, perhaps where we keep ASCAT

Reply:
Updated. See comment below